### DISCUSSION TITLE
I want to change the port from 8000. How can I do that?

### DISCUSSION THREAD

**Discussion by aliahadmd** [mythrantic/ollama-docker#10] [General]
![image](https://github.com/valiantlynx/ollama-docker/assets/90032081/3aad380c-e271-4c0f-a6ee-d39bd6b3bccc)

**Comment by valiantlynx** [comment]
i think the easiest way it to just change all of em

**Comment by valiantlynx** [comment]
the latest commit is configured to be easy to change ports without conflicts. the issue is the respective services need spesific env variables when you change ports
